gdal/src/main/scala/geotrellis/gdal/GDALReader.scala

```diff
@@ -113,13 +113,13 @@ class GDALReader(val dataset: Dataset) {
         if(cellType.isUnsigned) {
           val ct = cellType match {
             case c: UByteCells with NoDataHandling => c
-            case _ => throw new Exception(s"CellType ${cellType} can't be casted to UByte.")
```

Comment:
I'm a little confused looking at existing code here. These exceptions might have been here before GT has had Unsigned CellType support. Would this just work if the check was removed altogether?

Reply:
This is used to cast CellType to UByteCellType.